SimpleSaver/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@kaynewang/simple-saver",
+  "version": "1.0.0",
+  "description": "A simple saving tool.",
+  "main": "./dist/index.js",
+  "module": "./dist/index.esm.js",
+  "files": [
+    "dist/",
+    "README.md"
+  ],
+  "keywords": [
+    "streamSaver",
+    "multi-file",
+    "zip"
+  ],
+  "scripts": {
+    "dev": "webpack-dev-server --config ./webpack.config.js",
+    "build": "rollup -c",
+    "lint": "eslint ./lib/**",
+    "test": "jest",
+    "prebuild": "npm run lint",
+    "pretest": "npm run build",
+    "preversion": "npm run test",
+    "prepublish": "npm run test",
+    "postbuild": "node ./post-build.js"
+  },
+  "author": {
+    "name": "KayneWang",
+    "email": "w.zengkai@gmail.com"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org/"
+  },
+  "repository": {
+    "url": "https://github.com/LancerComet/MyWebLibs",
+    "type": "git"
+  },
+  "license": "ISC",
```

Comment:
It's better to have a real license and a LICENSE file

Reply:
Thx dude, I have fixed it.